docs/UNDERLAYS.md

```diff
@@ -0,0 +1,15 @@
+# Underlays
```

Comment:
This is good information to document, but most of these underlays are not accessible by any contributor to this repo. Perhaps our [internal wiki](https://github.com/verily-src/verily-tanagra-backend/wiki) is a better place to store this info?

Reply:
moved